### PR TITLE
doc cleanup, add sample elf for examples

### PR DESCRIPTION
**Pull request by jeffnye-gh**
Moved the original read me to ESPERANTO_README.md
New README.md with updated docs

Switch cleanup in main
Add bmi_mm.bare.riscv and bmi_sanity.bare.riscv sample elfs Update the golden traces

### CODE REVIEW

File: tests/stf_gen/golden/golden.bz2
Comment (StanSofomoDev) [inline tests/stf_gen/golden/golden.bz2:1]: I don't think that this should be removed - doing that might be the reason why ad hoc stf_gen test is failing for you

**Review by StanSofomoDev — CHANGES_REQUESTED** [review]

File: ESPERANTO_README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+/* set loc counter */
+. = 0x20000000;
+```
+
+2. **Modify and build Bootrom**
+
+You need to  provide a custom bootrom via the `--bootrom` option to run dromajo with custom base address. Modify the Makefile in the bootrom subdirectory to your desired start address. Update the MEMORY_START variable in the Makefile, for example:
+
+```Makefile
+MEMORY_START=0x15000000
```
[the remark this answers is not in the document]
Comment (jeffnye-gh) [reply ESPERANTO_README.md:95]: fixed

File: ESPERANTO_README.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+
+# Dromajo - RISC-V Reference Model
+
+This source is derived from the original Esperanto source which was
+derived from Fabrice Bellard's RISCVEMU/TinyEMU.
+
+The original contents of the read me are found in ORIGINAL_README.md
```
[the remark this answers is not in the document]
Comment (jeffnye-gh) [reply ESPERANTO_README.md:7]: removed that paragraph, now starts with original Esperanto paragraph

File: tests/stf_gen/golden/golden.bz2
Comment (jeffnye-gh) [reply tests/stf_gen/golden/golden.bz2:1]: was not intended to be removed, added back.

this was not the reason ad hoc failed.

File: ESPERANTO_README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeffnye-gh) [reply ESPERANTO_README.md:7]: deleted that section, now begins with the original Esperanto paragraph

**Comment by StanSofomoDev** [comment]
@jeffnye-gh I think that you haven't pushed commit with recent fixes :)

**Comment by jeffnye-gh** [comment]
> @jeffnye-gh I think that you haven't pushed commit with recent fixes :)

Check again, it was pushed 10 hours ago, I just did a fresh pull and switch and the changes are there.

commit 5ef0ea457941882159944362e1c397cc9a413e8a (HEAD -> jeffnye/doc_cleanup_for_ext_release, origin/jeffnye/doc_cleanup_for_ext_release)
Author: Jeff Nye <jeff@www.condorcomputing.com>
Date:   Wed Sep 25 21:37:06 2024 -0500

    review fixes

**Pull request merged**